src/modules/parseSupersetEmbeddedDashboardFieldValues.js

```diff
@@ -0,0 +1,17 @@
+export const parseSupersetEmbeddedDashboardFieldValues = (values) => ({
```

Comment:
Should `values` have a default {} ?

Reply:
I think not. There is no valid use-case for calling this function without a `values` argument. I'd rather have this function crash than it silently producing unexpected results.

(Looking forward to us switching to TypeScript, which will allow us to differentiate between required and optional parameters. Obviously we could then still have debates on wether a parameter should be optional with a default, or required, but at least my intentions would be 100% clear to you when you read the code)